### ISSUE TITLE
Embedded JavaScript

### ISSUE THREAD

**Issue by con-ssc** [Microsoft/vscode-ios-web-debug#12]
I have an html page with embedded javascript in a script tag. Can I debug this with the extension also?

**Comment by auchenberg** [comment]
At this point in time none of our VS Code JavaScript debuggers supports inline JS.

**Comment by con-ssc** [comment]
:( What would be a possible workaround?`Separating the js?

**Comment by auchenberg** [comment]
Yes, it would be to put the script it's own `.js` file and include it. It needs to be a file you can open independently in your editor.
